### PR TITLE
Update user guide on confirmation prompt and edit

### PR DESCRIPTION
**Pull request by marcustw**
Edit user guide

### CODE REVIEW

**Review by nweiyue — APPROVED** [review]
LGTM

File: docs/UserGuide.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ ATAS is a **desktop app for managing students’ particulars, optimized for use
   * [Listing all students](#list)
   * [Locating students by name](#find)
   * [Deleting a student](#delete)
+  * [Edit a student](#edit)
```
Comment (erisjacey) [inline docs/UserGuide.md]: Shouldn't this be renamed to `Editing a student` to fit the rest of the item naming?

**Review by erisjacey — COMMENTED** [review]
LGTM! Just one small nit.

**Review by erisjacey — APPROVED** [review]
LGTM

**Pull request merged**